src/server/worldserver/worldserver.conf.dist

```diff
@@ -3530,6 +3530,39 @@ Calculate.Gameoject.Zone.Area.Data = 0
 
 LFG.Location.All = 0
 
+#    Dungeon Access Requirements Print modes
+#
+#     Select the preferred format to display information to the player who cannot enter a portal dungeon because when has not met the access requirements:
+#         0 = Display no extra information, only "Requirements  not met"
+#         1 = Display only one requirement at a time (BlizzLike, like in the LFG interface)
+#         2 = Display detailed requirements, all at once, with clickable links
+#        Default: 1
+#
+DungeonAccessRequirements.PrintMode = 1
+
```

Comment:
```suggestion

#
#    DungeonAccessRequirements.PrintMode
#
#        Description: Select the preferred format to display information to the player who cannot enter a portal dungeon because when has not met the access requirements:
#        Default:     1 - (Display only one requirement at a time (BlizzLike, like in the LFG interface))
#                     0 - (Display no extra information, only "Requirements  not met")
#                     2 - (Display detailed requirements, all at once, with clickable links)
#

DungeonAccessRequirements.PrintMode = 1

```

Reply:
Add again, cuz you reverted it